packages/babel-standalone/src/index.js

```diff
@@ -100,11 +100,11 @@ function processOptions(options) {
 }
 
 export function transform(code: string, options: Object) {
```

Comment:
As we disallow `babel.transform` without callback in Babel 8. The `@babel/standalone` API becomes inconsistent with `@babel/core`. I am open to either align `@babel/standalone` to `@babel/core` (export transform{,Sync,Async}) or leave it as-is.

Reply:
+1 for aligning it.